server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider using uppercase for constant variables, like HOSTNAME should be renamed to HOSTNAME.